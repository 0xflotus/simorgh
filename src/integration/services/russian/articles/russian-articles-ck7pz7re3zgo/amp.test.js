/**
 * @service russian
 * @pathname /russian/articles/ck7pz7re3zgo
 */

import runAmpTests from '../../../../pages/articles/ampTests';

describe('AMP russian articles', runAmpTests);
