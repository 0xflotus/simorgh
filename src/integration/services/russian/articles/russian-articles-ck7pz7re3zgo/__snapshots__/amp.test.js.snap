// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP russian articles A11y I can see the skip to content link 1`] = `"Перейти к содержанию"`;

exports[`AMP russian articles AMP Consent config 1`] = `
Object {
  "consents": Object {
    "user-consent": Object {
      "promptIfUnknownForGeoGroup": "eea",
      "promptUI": "consent-prompt",
    },
  },
  "policy": Object {
    "default": Object {
      "timeout": Object {
        "fallbackAction": "reject",
        "seconds": 0,
      },
      "waitFor": Object {
        "user-consent": Array [],
      },
    },
  },
}
`;

exports[`AMP russian articles AMP Geo config 1`] = `
Object {
  "ISOCountryGroups": Object {
    "eea": Array [
      "at",
      "ax",
      "be",
      "bg",
      "bl",
      "cy",
      "cz",
      "de",
      "dk",
      "ea",
      "ee",
      "es",
      "fi",
      "fr",
      "gb",
      "gf",
      "gg",
      "gi",
      "gp",
      "gr",
      "hr",
      "hu",
      "ic",
      "ie",
      "im",
      "is",
      "it",
      "je",
      "li",
      "lt",
      "lu",
      "lv",
      "mf",
      "mq",
      "mt",
      "nc",
      "nl",
      "no",
      "pf",
      "pl",
      "pm",
      "pt",
      "re",
      "ro",
      "se",
      "si",
      "sj",
      "sk",
      "tf",
      "va",
      "wf",
      "yt",
    ],
  },
}
`;

exports[`AMP russian articles Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/russian"`;

exports[`AMP russian articles Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. Би-би-си на несет ответственности за содержание других сайтов. Познакомьтесь с нашими правилами внешних ссылок."`;

exports[`AMP russian articles Header I can see the branding 1`] = `"BBC News, Русская служба"`;

exports[`AMP russian articles I can see an image with a caption 1`] = `"https://ichef.bbci.co.uk/news/640/cpsdevpb/40ef/test/09d6c9a0-f63d-11e9-9afe-1371928331c5.jpg"`;

exports[`AMP russian articles I can see an image with a caption 2`] = `"Подпись к фото, Это тестовое изображение, авторское право BBC, показывает карту Франции. Изображение находится в первых трех блоках и имеет эту подпись."`;

exports[`AMP russian articles SEO Canonical link 1`] = `"http://localhost:7080/russian/articles/ck7pz7re3zgo"`;

exports[`AMP russian articles SEO Dir attribute 1`] = `"ltr"`;

exports[`AMP russian articles SEO FB admins 1`] = `"100004154058350"`;

exports[`AMP russian articles SEO FB app ID 1`] = `"1609039196070050"`;

exports[`AMP russian articles SEO Heading level 1 1`] = `"Это заголовок этой тестовой статьи"`;

exports[`AMP russian articles SEO Lang attribute 1`] = `"ru"`;

exports[`AMP russian articles SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "Article",
  "about": Array [
    Object {
      "@type": "Thing",
      "name": "Russia",
    },
  ],
  "author": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/russian.png",
      "width": 1024,
    },
    "name": "BBC News Русская служба",
    "noBylinesPolicy": "https://www.bbc.com/russian/institutional-50098149#authorexpertise",
  },
  "dateModified": "2019-07-12T14:29:44.981Z",
  "datePublished": "2019-07-12T14:29:44.981Z",
  "headline": "Пугало полицейского с камерой следит за водителями под Ливерпулем",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/russian.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/russian/articles/ck7pz7re3zgo",
    "@type": "WebPage",
    "name": "Пугало полицейского с камерой следит за водителями под Ливерпулем",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/russian.png",
      "width": 1024,
    },
    "name": "BBC News Русская служба",
    "publishingPrinciples": "https://www.bbc.com/russian/institutional-50098149",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/russian.png",
  "url": "http://localhost:7080/russian/articles/ck7pz7re3zgo",
}
`;

exports[`AMP russian articles SEO OG description 1`] = `"Пугало полицейского с камерой следит за водителями под Ливерпулем"`;

exports[`AMP russian articles SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/russian.png"`;

exports[`AMP russian articles SEO OG image alt 1`] = `"BBC News Русская служба"`;

exports[`AMP russian articles SEO OG locale 1`] = `"ru-RU"`;

exports[`AMP russian articles SEO OG site name 1`] = `"BBC News Русская служба"`;

exports[`AMP russian articles SEO OG title 1`] = `"Пугало полицейского с камерой следит за водителями под Ливерпулем - BBC News Русская служба"`;

exports[`AMP russian articles SEO OG type 1`] = `"article"`;

exports[`AMP russian articles SEO OG url 1`] = `"http://localhost:7080/russian/articles/ck7pz7re3zgo"`;

exports[`AMP russian articles SEO Page title 1`] = `"Пугало полицейского с камерой следит за водителями под Ливерпулем - BBC News Русская служба"`;

exports[`AMP russian articles SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`AMP russian articles SEO Twitter card 1`] = `"summary_large_image"`;

exports[`AMP russian articles SEO Twitter creator 1`] = `"@bbcrussian"`;

exports[`AMP russian articles SEO Twitter description 1`] = `"Пугало полицейского с камерой следит за водителями под Ливерпулем"`;

exports[`AMP russian articles SEO Twitter image alt 1`] = `"BBC News Русская служба"`;

exports[`AMP russian articles SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/russian.png"`;

exports[`AMP russian articles SEO Twitter site 1`] = `"@bbcrussian"`;

exports[`AMP russian articles SEO Twitter title 1`] = `"Пугало полицейского с камерой следит за водителями под Ливерпулем - BBC News Русская служба"`;
